frontend/src/formdescriptioneditor/TextfieldWidget.tsx

```diff
@@ -10,28 +10,50 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
+import { makeStyles } from '@material-ui/core/styles';
 import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
 import { WidgetProps } from 'formdescriptioneditor/WidgetEntry.types';
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+
+const useStyles = makeStyles(() => ({
+  selected: {
+    color: 'var(--blue-lagoon)',
```

Comment:
use `theme.palette.primary.main` instead of 'var(--blue-lagoon)'

Reply:
Fixed